### PR TITLE
Maddagada

### PR DESCRIPTION
**Pull request by maddagada**
Added User Interface for login page. @jfranco1701 Please Review it and let me know your suggestions if any.

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ You'll want to downlaod and install PostgreSQL onto your system through their [s
 
 Launch the PostgreSQL manager (I do it though the `psql` command in my terminal) and create the database by:
 
-    create database stem_ecosystem;
+    CREATE database stem_ecosystem;
```
Comment (codykaup) [inline SETUP.md]: Did you run into issues with this step? All caps is a SQL convention preference but it shouldn't throw errors anywhere.

File: src/frontend/src/app/login/login.component.html [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+}
+
+/* Change styles for span and cancel button on extra small screens */
+@media screen and (max-width: 300px) {
+  span.psw {
+     display: block;
+     float: none;
+  }
+  }
+</style>
```
Comment (codykaup) [inline src/frontend/src/app/login/login.component.html]: We'll probably want to throw all the styling into the primary .scss file for each component so similar parts stay together.

File: src/frontend/src/app/login/login.component.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.container {
+    margin-top:25px;
+  }
+  
+  .login {
```
Comment (codykaup) [inline src/frontend/src/app/login/login.component.scss]: It's probably minor OCD but it'd be better to have these lines match up by sliding all the `.login` pieces to the left.

File: src/frontend/src/app/login/login.component.html (hunk 2 above)
Comment (jfranco1701) [reply src/frontend/src/app/login/login.component.html]: Yes, all the css should be in the .scss file.

File: src/frontend/src/app/login/login.component.scss (hunk 3 above)
Comment (jfranco1701) [reply src/frontend/src/app/login/login.component.scss]: The html file should not have a html, head, body, etc tags.  This is all in the index.html page out in the root of the project.  Just start with your div.

Comment (jfranco1701) [reply src/frontend/src/app/login/login.component.scss]: The login page is also not making use of material

File: SETUP.md (hunk 1 above)
Comment (maddagada) [reply SETUP.md]: In windows, it worked only when I used uppercase letters and so I changed it.

**Pull request merged**